init.el

```diff
@@ -64,6 +64,12 @@ by Prelude.")
   "This folder stores all the automatically generated save/history-files.")
 (defvar prelude-modules-file (expand-file-name "prelude-modules.el" prelude-dir)
   "This files contains a list of modules that will be loaded by Prelude.")
+(defvar prelude-package-archives nil
```

Comment:
Do we really need those extra vars. Seems to me someone can simply add the extra repos and set the pinned packages directly...